inc/ticket.class.php

```diff
@@ -610,12 +610,11 @@ function getTabNameForItem(CommonGLPI $item, $withtemplate=0) {
                $ong[2] = _n('Solution', 'Solutions', 1);
             }
             // enquete si statut clos
-            if (($item->fields['status'] == self::CLOSED)
-                && Session::haveRight('ticket', Ticket::SURVEY)){
-               $satisfaction = new TicketSatisfaction();
-               if ($satisfaction->getFromDB($item->getID())) {
-                  $ong[3] = __('Satisfaction');
-               }
+            $satisfaction = new TicketSatisfaction();
+            if ($satisfaction->getFromDB($item->getID())
+                     && $item->fields['status'] == self::CLOSED
+                     && $satisfaction->canUpdateItem()){
```

Comment:
And it's even better, as like this the form can be viewed event if the right to update is not given